lambdas/ec2/README.md

```diff
@@ -0,0 +1,42 @@
+# AWS EC2 Reaper 
+
+This AWS EC2 Reaper works using tags set on the instance itself. The Reaper is composed of two AWS Lambdas: the "Schema Enforcer" and the "Terminator". The Schema Enforcer ensures that all new instances have a valid `termination_date` tag set when the EC2 resource is created, and the Terminator runs periodically to delete all instances past their `termination_date`.
+
+## Rules and Usage
+
+1. The Schema Enforcer ensures that newly created EC2 instances have a valid `termination_date` tag set. If a valid `termination_date` is not set within 4 minutes, the Schema Enforcer will terminate the instance and raise an exception. The `termination_date` must be a valid IS0 8601 value with a UTC offset defined.
+    * Instead of setting the `termination_date` directly, you can set a `lifetime` tag to specify how long the instance should live. A valid value is a string of of an integer value with a 1 letter unit of w(weeks), d(days), h(hours). For example, `1w` is 1 week, `2d` is 2 days, and `3h` is 3 hours. The Schema Enforcer will caculcate a future date based upon the `lifetime` tag and set a new `termination_date` on the instance.
```

Comment:
> the Schema Enforcer will terminate the instance and raise an exception

What does "raise an exception" mean for a user of AWS?

Overall, is this "Rules and Usage" section targeted at a user or a developer?  Seems like it's trying to be for the user, and the mention of exceptions is throwing me off because I don't know what behavior the user would experience because of a thrown exception.